packages/starter/vanilla-ui-starter/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Crossmint pay button</title>
+    <script src="../../ui/vanilla-ui/lib/index.global.js"></script>
```

Comment:
If a customer copy pastes the starter this import wont work, no?

Shouldnt we import the package via npm and  import it here somehow differently?

Reply:
Forgot to mention!! Probably here we will have an CDN link like https://unpkg.com/browse/@crossmint/client-sdk-vanilla-ui@0.0.1-alpha.0/lib/CrossmintPayButton.js

But since this is not deployed yet, we cant place it there 😄